test-suite/shared/src/test/require-jdk8/org/scalajs/testsuite/javalib/time/TemporalAccessorTest.scala

```diff
@@ -7,8 +7,8 @@ import org.junit.Test
 import org.junit.Assert._
 import org.scalajs.testsuite.utils.AssertThrows._
 
-abstract class TemporalAccessorTest {
-  val samples: Seq[TemporalAccessor]
+abstract class TemporalAccessorTest[A <: TemporalAccessor] {
+  val samples: Seq[A]
```

Comment:
Sure, will rename it. Since I'm also removing the `test_` prefix from the `test_range` method in the `TemporalAccessorTest` class, probably it would make sense to remove that prefix in the renaming methods (including the abstract `TemporalTest` as well), just to help _keep the house clean_ if that sounds good to you too.